cmd/root.go

```diff
@@ -0,0 +1,60 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/spf13/viper"
+)
+
+var cfgFile string
+
+// rootCmd represents the base command when called without any subcommands
+var rootCmd = &cobra.Command{
+	Use:   "crda",
+	Short: "crda is cli tool to interact with CRDA Platform.",
+	Long:  `crda is cli tool to interact with CRDA Platform.`,
+	Args:  cobra.ExactValidArgs(1),
+}
+
+// Execute adds all child commands to the root command and sets flags appropriately.
+// This is called by main.main(). It only needs to happen once to the rootCmd.
+func Execute() {
+	if err := rootCmd.Execute(); err != nil {
+		fmt.Println(err)
+		os.Exit(1)
+	}
+}
+
+func init() {
+	cobra.OnInitialize(initConfig)
+
+	rootCmd.PersistentFlags().StringVar(&cfgFile, "config", "", "config file (default is $PROJECT_ROOT/config.yaml)")
+
+	rootCmd.Flags().BoolP("toggle", "t", false, "Help message for toggle")
```

Comment:
You probably need to add a verbose flag for more detailed logging.

Reply:
added `crdaDebug` env variable

Reply:
Added new Flag `--debug`